index.js

```diff
@@ -338,3 +338,24 @@ Phantasma.prototype.content = function (html) {
     }
   });
 };
+Phantasma.prototype.extractDomElement = function (selector, path) {
+    var self = this;
+    return this.evaluate(function (selector) {
+        return document.querySelector(selector).getBoundingClientRect();
+    }, selector).then(function (value) {
+        self.page.set('clipRect', value);
+        return new self.promise(function (resolve, reject) {
+            self.page.render(path, resolve);
+            self.restoreRect();
```

Comment:
It has to modify the "rect" to the dom element rect options, modifying the viewport() of the selector so that it can make a screenshot of it. After that it takes the viewport() of the html witch should be the whole page, that's where restoreRect() comes in play. 
If you have other suggestions, I'm lessening.